src/components/NavigationMailbox.vue

```diff
@@ -197,6 +197,7 @@ export default {
 			renameInput: false,
 			mailboxName: this.mailbox.displayName,
 			showMoveModal: false,
+			hasDelimiter: this.mailbox.delimiter !== null,
```

Comment:
```suggestion
			hasDelimiter: !this.mailbox.delimiter,
```

then it works also with `undefined` :wink: